Pinta.Resources/icons/hicolor/scalable/readme.md

```diff
@@ -0,0 +1,20 @@
+## Required GTK3 conventions for SVG icons:
+
+- Icon names must end in `-symbolic.svg`
+- SVG elements must contain `fill="#bebebe"`
+  - This `fill` will automatically be updated by GTK3 to an appropriate color to match the user's theme settings.
+
+## Attribution
+
+Some icons are from the following sets:
```

Comment:
I think there are also some icon credits in the About dialog, so maybe these can be merged in there? I'm not sure if the existing credits are for some of the icons removed in this change :)

Reply:
Added these sets to the existing list in the About box.  The remaining old icons we use are still a mix of all the existing sets, so I don't think we can remove those credits until all the `.png` files are replaced.

I also removed the "Powered by Mono" part of the About box while making this change, since that is no longer accurate.  😁